src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -4,47 +4,38 @@
 import java.time.format.DateTimeFormatter;
 
 public class SalaryInfo {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private final int dataIndex = 0;
+    private final int nameIndex = 1;
+    private final int indexOfHour = 2;
+    private final int salaryIndex = 3;
```

Comment:
these are constants, let's give them proper style.
-  They must be static 
- their names must be uppercase